backend/server/static/api/meetingbook.yaml

```diff
@@ -58,12 +58,72 @@ paths:
           description: "Unauthorized"
   /schedule:
     get:
+      parameters:
+        - in: query
+          name: admin_id
```

Comment:
I am not sure this is a good idea :) 
It is a security issue cause I can get someone's schedule guessing the id.
I would remove this parameter.

**How to get admin?**

Easy! Since this route is private and secured by Basic Auth you can always get the credentials (admin email for example) from Authorization header. You check that header anyway :)

Reply:
agree